src/middlewared/middlewared/plugins/chart_releases_linux/acl.py

```diff
@@ -0,0 +1,15 @@
+from middlewared.service import private, Service
+
+
+class ChartReleaseService(Service):
+
+    class Config:
+        namespace = 'chart.release'
+
+    @private
+    async def apply_acls(self, acls_to_apply):
+        bulk_job = await self.middleware.call(
+            'core.bulk', 'filesystem.add_to_acl', [acls_to_apply[acl_path] for acl_path in acls_to_apply],
+        )
+        await bulk_job.wait()
+        # TODO: Should we raise an error?
```

Comment:
There are a few cases where this may happen:
1) Invalid payload
2) Permissions on the parent path would prevent the ACL from granting intended permissions (e.g. parent path has mode of 0700)
3) Dataset has ACLs disabled
4) Dataset has POSIX1E ACL type and payload specifies FULL_CONTROL

It would be helpful to give administrator some feedback here.